src/main/scala/com/danielasfregola/twitter4s/http/oauth/OAuth1Provider.scala

```diff
@@ -80,7 +80,9 @@ private[twitter4s] class OAuth1Provider(consumerToken: ConsumerToken, accessToke
     implicit val ec = materializer.executionContext
     extractRequestBody.map { body =>
       val cleanBody = body.replace("+", "%20")
-      if (cleanBody.nonEmpty) {
+      if (cleanBody.nonEmpty && request.entity
+            .getContentType()
+            .mediaType == MediaTypes.`application/x-www-form-urlencoded`) {
```

Comment:
Why do we need `request.entity
            .getContentType()
            .mediaType == MediaTypes.application/x-www-form-urlencoded`?

Reply:
Because, in new api, if we send POST request for creation new direct message, we cann't parameters in the url, we send only body as json. And this function cann't calculate Map, because `cleanBody` have size = 1. May be, I can change Map calculation to Try.toOption